flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/plan/nodes/exec/stream/StreamExecCalc.java

```diff
@@ -42,6 +42,7 @@
 @ExecNodeMetadata(
         name = "stream-exec-calc",
         version = 1,
+        consumedOptions = {"table.optimizer.simplify-operator-name-enabled"},
```

Comment:
I see this one is used everywhere, does it makes sense to assume it by default?

Reply:
Or I can somehow add it automatically in the annotation list for all nodes, since it's used in `ExecNodeBase`.

Reply:
TBH we should rediscuss the key `table.optimizer.simplify-operator-name-enabled` this has nothing to do with optimizer. The optimization is already done at this point. I would vote to call it `table.exec.simplify-operator-name`.

Reply:
makes sense!